source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1289,7 +1289,7 @@ namespace
     // already been created.
     PvtPrim* readLookGroup(const LookGroupPtr& src, PvtStage* stage, PvtRenamingMapper& mapper)
     {
-        const string LIST_SEPARATOR(",");
+        const string LIST_SEPARATOR(", ");
```

Comment:
Not sure why you need the space. splitstring should handle with or without a space after. 
If there is no space, I think this will fail reading won't it ?

Reply:
My understanding of splitstring is that it requires all of the identifiers to split by which in this case is space and comma. I was running into issues with only comma. Typically we pass ARRAY_VALID_SEPARATORS or ARRAY_PREFERRED_SEPARATOR to splitstring which has a comma and space in it.

Reply:
Agreed, let's change this to use the existing constants and remove this one. Thanks.